sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/Expression.scala

```diff
@@ -90,7 +90,7 @@ abstract class Expression extends TreeNode[Expression] {
     val primitive = ctx.freshName("primitive")
     val ve = GeneratedExpressionCode("", isNull, primitive)
     ve.code = genCode(ctx, ve)
-    ve
```

Comment:
I think it is probably worth it.  When codegen bugs pop-up in the real world this makes it possible to debug, where as otherwise the code is super hard to follow.  I tested on a pretty deeply nested expression and the code size only went from 25kb to 50kb.  I'd imagine the parser is pretty good at skipping over comments.